core/models.py

```diff
@@ -0,0 +1,73 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.translation import gettext_lazy as _
+
+class ChecklistQuestionAnswer(models.IntegerChoices):
+    NO = 0, _("No")
+    YES = 1, _("Yes")
+    NOT_RELEVANT = 2, _("This isn't relevant to me")
+
+class NoteType(models.IntegerChoices):
+    PA = 0, _("Pathway Plan PA")
+    MANAGER = 1, _("Pathway Plan Manager")
+
+class PersonalAdvisor(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    manager = models.ForeignKey(User, related_name="personal_advisors", null=True, on_delete=models.SET_NULL)
+    is_manager = models.BooleanField()
+
+class YoungPerson(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    personal_advisors = models.ManyToManyField(PersonalAdvisor, related_name="young_persons")
+
+class Goal(models.Model):
+    young_person = models.ForeignKey(YoungPerson, on_delete=models.CASCADE, related_name="goals")
```

Comment:
it seems that `created` and `last_updated` are repeated across some models. What do you think of creating a `MetadataMixin` that both those models inherit from? @kws made one for HFU and I think it makes the code cleaner:

```
class MetadataMixin(models.Model):
    created_at = models.DateTimeField(auto_now_add=True, editable=False)
    updated_at = models.DateTimeField(auto_now=True, editable=False)

    class Meta:
        abstract = True
```

and then:
```
class Goal(MetadataMixin, models.Model):
    ....
```